CONTRIBUTING.md

```diff
@@ -1,18 +1,19 @@
 # Contributing
-The PostGraphQL project welcomes all contributors, we want to invest in you so you can invest back into PostGraphQL. Together we can make great software that enables developers to build powerful applications in much less time then would previously be taken. This document aims to help you get started contributing to the project.
+The PostGraphQL project welcomes all contributors, we want to invest in you so you can invest back into PostGraphQL. Together we can make great software that enables developers to build powerful applications in much less time then would previously have been taken. This document aims to help you get started contributing to the project.
 
-As PostGraphQL may be a piece of critical infrastructure in your app, it is only fair to run the project as [OPEN Open Source](http://openopensource.org/). If contribute meaningful work to the PostGraphQL project you will be made a collaborator which allows you to create new branches and approve pull requests.
+As PostGraphQL may be a piece of critical infrastructure in your app, it is only fair to run the project as [OPEN Open Source](http://openopensource.org/). If you contribute meaningful work to the PostGraphQL project you will be made a collaborator which allows you to create new branches and approve pull requests.
 
-The codebase is documented via READMEs throughout the src folder heirarchy, starting with [`src/README.md`](src/README.md). Contributions are also encouraged where these files are missing or inadequate.
+The codebase is documented via READMEs throughout the `src` folder heirarchy, starting with [`src/README.md`](src/README.md). Contributions are also encouraged where these files are missing or inadequate.
 
 To get started hacking on the codebase, make sure Postgres is listening on `localhost:5432` and then run:
 
 ```bash
-scripts/run-kitchen-sink-schema
-scripts/dev
+createdb postgraphql
+scripts/run-kitchen-sink-sql postgraphql
+scripts/dev -c pg://localhost/postgraphql
```

Comment:
How about we add a note instead that this is a valid alternative? Something like:

```
This will put the schemas into your default database. If you would prefer to put the schemas in another database pass a connection argument like so:

createdb postgraphql
scripts/run-kitchen-sink-sql postgres://localhost:5432/postgraphql
scripts/dev -c postgres://localhost:5432/postgraphql
```